src/main/java/kostka/moviecatalog/service/ExternalShopService.java

```diff
@@ -45,25 +44,25 @@
     private CommunicationService communicationService;
     private UserService userService;
     private CacheService cacheService;
-    private ObjectMapper mapper;
+    private JsonConvertService jsonConvertService;
 
     @Autowired
     public ExternalShopService(
             final CommunicationService communicationService,
             final DbMovieService dbMovieService,
             final UserService userService,
             final CacheService cacheService,
-            final ObjectMapper mapper) {
+            final JsonConvertService jsonConvertService) {
         this.communicationService = communicationService;
         this.dbMovieService = dbMovieService;
         this.userService = userService;
         this.cacheService = cacheService;
-        this.mapper = mapper;
+        this.jsonConvertService = jsonConvertService;
     }
 
     @PostConstruct
     public void setUp() {
-        mapper.registerModule(new JavaTimeModule());
+        jsonConvertService.registerMapperModule(new JavaTimeModule());
```

Comment:
minor: přemýšlím jen nad jednou věcí. Co když nějaká další služba bude JavaTimeModule taky potřebovat. Může/musí tu registraci dělat znovu?
Jestli by v budoucnu nestálo za to, registrovat to nějak centrálně, např v kódu pro inicializaci jsonConvertService beany samotné.